src/backend/application_server.py

```diff
@@ -0,0 +1,133 @@
+
+"""
+Module defining our flask application server.
+"""
+
+import os
+import logging
+from PIL import Image
+from pathlib import Path
+import torch
+import torchvision.transforms as transforms
+from flask import Flask, flash, request, redirect, render_template,\
+                  send_from_directory
+from werkzeug.utils import secure_filename
+from gan import Generator
+from torchvision.utils import save_image
+
+UPLOAD_FOLDER = Path.cwd() / 'uploads/'
+Path(UPLOAD_FOLDER).mkdir(exist_ok=True)
+RESPONSE_FOLDER = Path.cwd() / 'response/'
+Path(RESPONSE_FOLDER).mkdir(exist_ok=True)
+ALLOWED_EXTENSIONS = {'bmp', 'png', 'jpg', 'jpeg', 'ppm', 'pgm', 'tif'}
+
+app = Flask(__name__)
+app.secret_key = b'MBWUdbxX;>]vrTL'
+app.config['MAX_CONTENT_LENGTH'] = 50 * 1024 * 1024
```

Comment:
What's the origin of those constants?

Reply:
This corresponds to a randomly set limit of 50 megabytes.